src/GeneralStateTestsFiller/stSolidityTest/ByZeroFiller.json

```diff
@@ -63,13 +53,12 @@
         },
         "//comment" : "multiple transaction description that will be applied on a pre state",
         "transaction" : {
-            "//comment" : "foreach d in `data` { foreach g in `gasLimit` { foreach v in `value`  {  pre.applyTransaction(d, g, v)  }}}",
-            "//comment" : "each element of the data array is treated same as account's code field. You could use LLL here",
+            "//comment" : "store 1 at the result of {DIV,SDIV,MOD,SMOD}(1,0)",
             "data" : [
-              "0x6001600004600055",
-              "0x6001600005600055",
-              "0x6001600006600055",
-              "0x6001600007600055"
+              "0x6001600160000455",
```

Comment:
I get this code from lllc  "[[1]] (div 1 0)" -> "600060010460015500"

Reply:
you could also use LLLC in transaction data

Reply:
> I get this code from lllc "[[0]] (div 1 0)" -> "600060010460005500"

That has different behaviour. I am storing the number 1 at storage location `div(1,0)`

Reply:
lllc "[[ (div 1 0) ]] 1" -> "600160006001045500"  is also different
maybe just to be safe you could use lllc in transaction data directly?